pkg/converter/pod_converter_test.go

```diff
@@ -75,15 +78,19 @@ var _ = Describe("PodConverter", func() {
 			},
 		}
 
-		wepData, err := c.Convert(&pod)
+		wepDatas, err := c.Convert(&pod)
+
 		By("not generating a conversion error", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		Expect(len(wepDatas)).Should(Equal(1))
+		wepData := wepDatas[0]
```

Comment:
Might be nice to have a test for multiple more than 1 wepData

Reply:
This PR is slightly misleading, for OS there will actually never be more than one, but OS libcalico-go using the interface of returning multiple WEPs so it will be compatible with private. The tests will definitely be there for private.